### PR TITLE
Issue 3 : Publish WHL for Python 3.10 targeting manylinux_2_31

### LINKED ISSUE #3: Publish WHL for Python 3.10 targeting manylinux_2_31
**Problem description**
Publish WHL for Python 3.10 targeting manylinux_2_31

**Problem location**
Currently python 3.10 is unable to download pravega-client unless the baseos is windows/mac/Ubuntu 22.04

### PR DESCRIPTION
**Pull request by Bhupender-Y**
**Change log description**
Publish WHL for Python 3.10 targeting manylinux_2_31.

**Purpose of the change**
Fixes #3  

**What the code does**
Release script will publish WHL for Python 3.10 targeting manylinux_2_31

**How to verify it**
WHL for Python 3.10 targeting manylinux_2_31 should be published on release.

### CODE REVIEW

File: .github/workflows/build_wheel.py [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+    command = [
+        "maturin",
+        "build",
+        "--release",
+        "--compatibility=manylinux_2_35",
+        "--interpreter",
+        sys.executable,
+    ]
+subprocess.run(command, check=True)
+os.chdir("..")
```
Comment (AnkurAnand11) [inline .github/workflows/build_wheel.py]: This line (line 29) can be removed as it is causing the build issue of FileNotFound Error for python 3.8 and earlier.

File: .github/workflows/tagPublish.yml [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+    name: Create Github release page
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          ref: ${{ github.event.release.tag_name }}
+      - name: Set release version
+        run: echo "RELEASE_VERSION=${GITHUB_REF#refs/*/}" >> $GITHUB_ENV
+      - name: Create Github release page
+        # Create the release page first or the nodejs artifacts upload will fail
```
Comment (AnkurAnand11) [inline .github/workflows/tagPublish.yml]: Correct the comment as we are not using nodejs here.

File: .github/workflows/tagPublish.yml [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+    runs-on: ${{ matrix.platform.os }}
+    strategy:
+      matrix:
+        python-version: ['3.7', '3.8', '3.9', '3.10', '3.11']
+        platform: [
+        { os: "ubuntu-latest",  python-architecture: "x64", rust-target: "x86_64-unknown-linux-gnu" },
+        { os: "macOS-latest",   python-architecture: "x64", rust-target: "x86_64-apple-darwin" },
+        { os: "windows-latest", python-architecture: "x64", rust-target: "x86_64-pc-windows-msvc" },
+        { os: "ubuntu-20.04",  python-architecture: "x64", rust-target: "x86_64-unknown-linux-gnu" },
+        # { os: "windows-latest", python-architecture: "x86", rust-target: "i686-pc-windows-msvc" },
```
Comment (AnkurAnand11) [inline .github/workflows/tagPublish.yml]: Remove this commented code if not required.

File: .github/workflows/build_wheel.py (hunk 1 above)
Comment (Bhupender-Y) [reply .github/workflows/build_wheel.py]: Done

File: .github/workflows/tagPublish.yml (hunk 2 above)
Comment (Bhupender-Y) [reply .github/workflows/tagPublish.yml]: Done.

File: .github/workflows/tagPublish.yml (hunk 3 above)
Comment (Bhupender-Y) [reply .github/workflows/tagPublish.yml]: Done.

File: .github/workflows/tagPublish.yml [hunk 4]
```diff
@@ -0,0 +1,85 @@
...
+      - name: install maturin
+        run: pip install 'maturin>=0.14,<0.15'
+      - name: build wheel
+        if: ${{ matrix.platform.os != 'ubuntu-20.04'}}
+        id: build_wheel
+        run: python -u .github/workflows/build_wheel.py
+      - name: build wheel 1
+        if: ${{ matrix.platform.os == 'ubuntu-20.04'}}
+        id: build_wheel1
+        run: python -u .github/workflows/build_wheel.py "maturin","build","--release","--compatibility=manylinux_2_31","--interpreter"
```
Comment (tkaitchuck) [inline .github/workflows/tagPublish.yml]: Here arguments are supplied , while above they are not. We should probably do it the same way for both code paths.

Comment (Bhupender-Y) [reply .github/workflows/tagPublish.yml]: Done

**Review by AnkurAnand11 — APPROVED** [review]
LGTM

File: .github/workflows/build_wheel.py [hunk 5]
```diff
@@ -0,0 +1,33 @@
...
+
+os.chdir("./")
+command = [
+    "maturin",
+    "build",
+    "--release",
+    "--interpreter",
+    sys.executable,
+]
+if len(sys.argv) > 1 :
```
Comment (tkaitchuck) [inline .github/workflows/build_wheel.py]: It looks like the argument is always passed. Can we removed this ?

**Review by tkaitchuck — APPROVED** [review]

File: .github/workflows/build_wheel.py (hunk 5 above)
Comment (Bhupender-Y) [reply .github/workflows/build_wheel.py]: Yes, i have removed the If condition.

**Review by tkaitchuck — APPROVED** [review]

**Pull request merged**